src/evict/evict_conn.c

```diff
@@ -233,7 +235,9 @@ __wt_evict_config(WT_SESSION_IMPL *session, const char *cfg[], bool reconfig)
 
 /*
  * __wt_evict_create --
- *     Initialize eviction.
+ *     This function sets up eviction's internal structures during `wiredtiger_open` to manage
+ *     eviction. It prepares the It must be called exactly once during `wiredtiger_open`, and must
+ *     be called before any eviction threads are spawned.
  */
 int
 __wt_evict_create(WT_SESSION_IMPL *session, const char *cfg[])
```

Comment:
Why is `cfg` here? The comment doesn't help me reason about that. 

Reply:
Updated in [b91a788](https://github.com/wiredtiger/wiredtiger/pull/11096/commits/b91a7887129f1d67c8c92c89cedae30fc04cb722)

Reply:
I think it's surprising (and Will agreed) that this calls the main eviction config function (even though you sort of tried to say that). I don't know if this is the magic solution, but I wanted to propose something rather than just nit-picking. I definitely don't feel strongly about this.

```
 *     Input parameter:
 *       `cfg[]`: A stack of configuration strings.  This is passed to `__evict_config` which configures
 *       all of eviction (ie `eviction.*` options) as part of the eviction setup process.
 *       
 * ```

Reply:
makes sense, updated in [11cba66](https://github.com/wiredtiger/wiredtiger/pull/11096/commits/11cba663c1856e3ede7e5dbdfae661ad04de53a6)

Reply:
Most of the create function call the config function internally in WT.